Week3/homework/2-step3.js

```diff
@@ -1,8 +1,41 @@
 'use strict';
```

Comment:
The (lack of proper) indentation in this file makes it difficult to quickly oversee its structure. VSCode can automatically do the proper formatting for you if you add some settings to it. See the [VSCode Tips](https://github.com/HackYourFuture/fundamentals/tree/master/VSCodeTips#customise-vscode-settings). **Please make sure you take action on this.**